gammapy/modeling/models/spectral.py

```diff
@@ -1979,11 +1967,17 @@ class NaimaSpectralModel(SpectralModel):
     tag = ["NaimaSpectralModel", "naima"]
 
     def __init__(
-        self, radiative_model, distance=1.0 * u.kpc, seed=None, nested_models=None
+        self,
+        radiative_model,
+        distance=1.0 * u.kpc,
+        seed=None,
+        nested_models=None,
+        use_cache=False,
```

Comment:
Please add an entry in the docstring for this option...if you are confident this is not an option that user would need regularly, we could also just not expose it.